packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -0,0 +1,149 @@
+    bannerId: 'choice-cards-banner-blue',
+    countryCode: 'GB',
+    content: {
+        mainContent: {
+            heading: <>Lend us a hand in 2023</>,
+            subheading: null,
+            paragraphs: [
+                <>
+                    Shareholders or billionaire owner, we report on world events with accuracy, free
+                    from political and commercial influence. And unlike many others, we’re committed
+                    to keeping our reporting open for all readers. Every contribution, however big
+                    or small, makes a difference.
+                </>,
+            ],
+            highlightedText: (
+                <>
+                    Support us from as little as £1. If you can, please consider supporting us with
+                    a regular amount each month. Thank you.
+                </>
+            ),
+            primaryCta: {
+                ctaText: 'Support us',
+                ctaUrl: 'https://support.theguardian.com/contribute',
+            },
+            secondaryCta: null,
+        },
+        mobileContent: {
+            heading: <>Lend us a hand in 2023</>,
+            subheading: null,
+            paragraphs: [
+                <>
+                    Shareholders or billionaire owner, we report on world events with accuracy, free
+                    from political and commercial influence. And unlike many others, we’re committed
+                    to keeping our reporting open for all readers. Every contribution, however big
+                    or small, makes a difference.
+                </>,
+            ],
+            highlightedText: (
+                <>
+                    Support us from as little as £1. If you can, please consider supporting us with
+                    a regular amount each month. Thank you.
+                </>
+            ),
+            primaryCta: {
+                ctaText: 'Support us',
+                ctaUrl: 'https://support.theguardian.com/contribute',
+            },
+            secondaryCta: null,
+        },
+    },
+    isSupporter: false,
+    tracking,
+    choiceCardAmounts: {
+        testName: 'Storybook_test',
+        variantName: 'Control',
+        amounts: {
+            ONE_OFF: {
+                amounts: [5, 10, 15, 20],
+                defaultAmount: 5,
+                hideChooseYourAmount: false,
+            },
+            MONTHLY: {
+                amounts: [6, 12, 18, 24],
+                defaultAmount: 12,
+                hideChooseYourAmount: true,
+            },
+            ANNUAL: {
+                amounts: [50, 100, 150, 200],
+                defaultAmount: 100,
+                hideChooseYourAmount: true,
+            },
+        },
+    },
+    backgroundColor: blueBannerBackgroundColor,
+    headingColor: blueBannerHeadingColor,
+    onCloseClick: () => null,
+};
+
+const { bannerId, backgroundColor, headingColor, ...rest } = ChoiceCardsBannerBlue.args;
```

Comment:
Not sure where this line used?

Reply:
Using destructuring here for the args so I can select which ones to use later on

Reply:
doh yes